assets/js/components/SettingsNotice/SettingsNoticeSingleRow.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Settings notice SettingsNoticeSingleRow component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { getIconFromType } from './utils';
+
+const SettingsNoticeSingleRow = ( {
+	notice,
+	type,
+	Icon,
+	LearnMore,
+} ) => {
+	return (
+		<div className="googlesitekit-settings-notice__row">
+			<div className="googlesitekit-settings-notice__icon">
+				{ Icon ? <Icon /> : getIconFromType( type ) }
+			</div>
+			<div className="googlesitekit-settings-notice__text">
+				{ notice }
+			</div>
+			{ LearnMore && (
+				<div>
```

Comment:
And here, let's remove this `div` if it is not needed.

Reply:
This should actually have had a class on it with some styles!  (just added this :+1: )